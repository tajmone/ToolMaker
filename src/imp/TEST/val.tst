%%SET FOO("Foo string")%%SET FIE(FIE,"Hej hopp")
%%(FOO),%%(FIE)
%%IF (FOO(%%(FOO)))
Yes! (FOO)
%%ELSE
No! (FOO)
%%ENDIF
%%IF (FIE(%%(FIE)))
Yes! (FIE)
%%ELSE
No! (FIE)
%%ENDIF
%%IF (FOO(*))
Yes! (FOO)
%%ELSE
No! (FOO)
%%ENDIF
%%IF (FIE(*))
Yes! (FIE)
%%ELSE
No! (FIE)
%%ENDIF
%%SET FOO(Fum)%%SET FIE(Fum2)
%%(FOO),%%(FIE)
%%IF (FOO(%%(FOO)))
Yes! (FOO)
%%ELSE
No! (FOO)
%%ENDIF
%%IF (FIE(%%(FIE)))
Yes! (FIE)
%%ELSE
No! (FIE)
%%ENDIF
