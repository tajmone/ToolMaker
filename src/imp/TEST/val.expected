########## val ##########
%%SET FOO("Foo string")%%SET FIE(FIE,"Hej hopp")
%%(FOO),%%(FIE)
%%IF (FOO(%%(FOO)))
Yes! (FOO)
%%ELSE
No! (FOO)
%%ENDIF
%%IF (FIE(%%(FIE)))
Yes! (FIE)
%%ELSE
No! (FIE)
%%ENDIF
%%IF (FOO(*))
Yes! (FOO)
%%ELSE
No! (FOO)
%%ENDIF
%%IF (FIE(*))
Yes! (FIE)
%%ELSE
No! (FIE)
%%ENDIF
%%SET FOO(Fum)%%SET FIE(Fum2)
%%(FOO),%%(FIE)
%%IF (FOO(%%(FOO)))
Yes! (FOO)
%%ELSE
No! (FOO)
%%ENDIF
%%IF (FIE(%%(FIE)))
Yes! (FIE)
%%ELSE
No! (FIE)
%%ENDIF
########## ../imp val.tst >val.tst.tmp; cat val.tst.tmp; rm val.tst.tmp ##########

Foo string,FIE,Hej hopp
Yes! (FOO)
Yes! (FIE)
Yes! (FOO)
Yes! (FIE)

Fum,Fum2
Yes! (FOO)
Yes! (FIE)
